rust/arrow/src/array/array_list.rs

```diff
@@ -711,8 +722,9 @@ mod tests {
         assert_eq!(1, sliced_array.offset());
         assert_eq!(2, sliced_array.null_count());
 
+        let null_bit_slice = BufferBitSliceMut::new(&mut null_bits);
```

Comment:
this might be able to be `BitBufferSlice` as it doesn't need to be mutable

Reply:
Nice catch :+1: